Example/RZTouchIDDemo.xcodeproj/xcuserdata/adamhowitt.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -2,4 +2,22 @@
 <Bucket
    type = "1"
    version = "2.0">
+   <Breakpoints>
+      <BreakpointProxy
+         BreakpointExtensionID = "Xcode.Breakpoint.FileBreakpoint">
+         <BreakpointContent
```

Comment:
Intentionally shared breakpoint? I think xcuserdata should be .gitignored, and everything in that folder should be removed from the repo.